plugin-core/src/test/java/appland/rpcService/DefaultAppLandJsonRpcServiceTest.java

```diff
@@ -2,18 +2,20 @@
 
 import appland.AppMapBaseTest;
 import appland.cli.AppLandCommandLineService;
-import appland.settings.AppMapApplicationSettings;
 import appland.settings.AppMapApplicationSettingsService;
-import appland.toolwindow.runtimeAnalysis.RuntimeAnalysisModel;
 import com.intellij.openapi.vfs.VirtualFile;
 import org.jetbrains.annotations.NotNull;
 import org.junit.Test;
 
 import java.util.Collection;
+import java.util.Map;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicReference;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.hasEntry;
```

Comment:
I didn't even know that we have Hamcrest in the classpath. I checked and it's a transitive dependency of a few of our other testing libraries (wiremock, for example). Using should be alright, I just haven't used it yet :)

Reply:
I didn't even know what Hamcrest was ;)
IntelliJ suggested this with AWS CodeWhisperer and it looked like exactly what I needed.